howso/utilities/feature_attributes/tests/test_infer_feature_attributes.py

```diff
@@ -88,6 +88,69 @@ def test_infer_features_attributes():
         assert expected_types[feature] == attributes['type']
 
 
+def test_infer_features_attributes_type_overrides():
+    """Test forced types for infer feature types for iris dataset."""
+    df = pd.read_csv(iris_path)
+
+    expected_types = {
+        "sepal_length": "nominal",
+        "sepal_width": "nominal",
+        "petal_length": "nominal",
+        "petal_width": "ordinal",
+        "class": "nominal"
+    }
+
+    features = infer_feature_attributes(
+        df,
+        type_overrides={
+            'nominal': ['sepal_length', 'sepal_width', 'petal_length'],
+            'ordinal': ['petal_width'],
+        }
+    )
+
+    for feature, attributes in features.items():
+        assert expected_types[feature] == attributes['type']
+
+
+def test_infer_features_attributes_type_overrides_ordinals():
+    """Test ordinal forced type, should have the same values as continuous outside of type and bounds."""
+    df = pd.read_csv(iris_path)
+
+    features = infer_feature_attributes(df)
+
+    features_overrides = infer_feature_attributes(
+        df,
+        type_overrides={
+            'ordinal': ['sepal_length', 'sepal_width', 'petal_length', 'petal_width']
+        }
+    )
+    for _, value in features.items():
+        value.pop('type', None)
+        value.pop('bounds', None)
+
+    for _, value in features_overrides.items():
+        value.pop('type', None)
+        value.pop('bounds', None)
+
+    assert features == features_overrides
+
+
+def test_infer_features_attributes_type_overrides_invalid():
```

Comment:
Might also be good to have a negative test to verify specifying the same feature for multiple types raises.